bedrock/redirects/redirects.py

```diff
@@ -2004,6 +2006,53 @@
     redirect('^status/minutes\.html$', 'https://wiki.mozilla.org/WeeklyUpdates'),
     redirect('^store$', 'https://store.mozilla.org'),
     redirect('^testdrivers$', 'http://wiki.mozilla.org/B2G_Testdrivers_Program'),
+    # bug 1124038
+    redirect(r'^thunderbird/organizations/(?:all-esr\.html|faq/?)$', 'https://www.thunderbird.net/organizations/'),
```

Comment:
We need to update the [tests for these redirects](https://github.com/mozilla/bedrock/blob/master/tests/redirects/map_globalconf.py). We should probably also add new tests for any new redirects added here, so we can be confident they're working as expected.

You can find info on how to run the redirect tests locally here: http://bedrock.readthedocs.io/en/latest/redirects.html#testing-redirects